docs/Guides/Serverless.md

```diff
@@ -22,6 +22,17 @@ application directly without the need for any additional tools, while the same
 code will be executed in your serverless platform of choice with an additional
 snippet of code.
 
+**Note**:
+
+Fastify will not allow adding more routes or registering plugins once `ready()` is called because [Fastify will "compile" all of your plugins and routes](https://github.com/fastify/fastify/issues/1771#issuecomment-515339364).
+
+In some serverless environments, code outside the exported handlers could be cached. This means you
+cannot add routes or register plugins inside the exported handlers because it will be added more than once
+to the cached Fastify instance.
+
+Either declare all your routes and register all your plugins outside the handlers
```

Comment:
I think this sentence is too vague: if I create a new fastify, how could I start it?